internal/rest/resources/tokens.go

```diff
@@ -43,6 +45,11 @@ func tokensPost(state state.State, r *http.Request) response.Response {
 		return response.BadRequest(err)
 	}
 
+	err = utils.ValidateFQDN(req.Name)
+	if err != nil {
```

Comment:
We usually don't use the pattern `if err := something(); err != nil {}` across the code base.

If there are longer lines it can impact the readability.

Reply:
alright, thanks for the context!

Reply:
alright, thanks for the context!